code/datums/actions/item_action.dm

```diff
@@ -24,6 +24,9 @@
 	else if((target && current_button.appearance_cache != item_target.appearance) || force) //replace with /ref comparison if this is not valid.
 		var/old_layer = item_target.layer
 		var/old_plane = item_target.plane
+		// reset the x & y offset so that item is aligned center
+		item_target.pixel_x = 0
+		item_target.pixel_y = 0		
```

Comment:
doublecheck that pistols with suppressors are still centered properly. i remember they had a weird offset thing they needed a couple years ago.

Reply:
![dreamseeker_G6fy1RXONF](https://user-images.githubusercontent.com/5195984/195763635-52eee41b-bea3-4c5f-9136-dd90ecc61cc9.png)

Pistols or suppressors have no `item_action` button.  I didn't see anything out of the ordinary when I tested just now to double check.